images/base/Dockerfile

```diff
@@ -95,7 +102,7 @@ RUN echo "Ensuring scripts are executable ..." \
       conntrack iptables iproute2 ethtool socat util-linux mount ebtables udev kmod \
       libseccomp2 pigz \
       bash ca-certificates curl rsync \
-      nfs-common \
+      nfs-common fuse3 \
```

Comment:
How large are these additions?

Reply:
looking at https://packages.debian.org/buster/fuse3 and dependencies, my estimate is around ~2.5MB, but would let @ktock confirm.

Reply:
Er sorry: this addition but also the rest of them: how much larger is the image overall?
Will investigate in my next pass at this

Reply:
Added size comparison (**Image size comparison** section) to the above PR description.

[summary] around 33 MB addition by fuse3 + `containerd-stargz-grpc` binary (stargz snapshotter) + negligable config file changes